supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -65,6 +68,10 @@ class JesApiQueryManager(val qps: Int Refined Positive, override val serviceRegi
       "and localize the files yourself."
   ))
 
+  private[statuspolling] lazy val nbWorkers = 2
+  private lazy val workerQps = refineV[Positive](Math.max(qps.value / nbWorkers, 1))
+    .getOrElse(throw new IllegalArgumentException("Programmer error ! Keep the min qps at 1 !"))
```

Comment:
Isn't the "or else" after the `Math.max` unreachable? It might not be a bad idea to actually throw on a nonsensical qps value assuming the failure mode is reasonable.